gists/pull-request-guidelines.md

```diff
@@ -0,0 +1,64 @@
+# Pull Request (PR) Guidelines
+
+Guidelines for creating pull requests at LaunchPad Lab.
+
+## Authors
+- Use a [pull request template](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository)
+  - These should come OOTB when starting a new project using the `client-template` or `rails-template`
+- Convert a PR to a [draft pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests) if it is not ready for review yet (useful for [long-running](#long-running) prs)
+- Link to the appropriate task(s) that the PR addresses
+- Specifically call out items / features that are out of scope
+- Provide screenshots and instructions to reproduce / find style issues if there are outstanding items that require design assistance
+- Add guidance about what the reviewer(s) should focus on
+- Document what the QA tester should be testing
+- Open the review app to make sure that there are no critical errors **before** assigning for review
+- Merge the PR after approval is received from all assigned reviewers
+  - This is **your** responsibility, not the reviewer's
+
+### Types of PRs
```

Comment:
I think this can be relegated to a separate Notes portion (like the presentation) or put in a quote block since it's not a firm expectation of authors nor a company-wide distinction.